charts/argo-workflows/Chart.yaml

```diff
@@ -3,7 +3,7 @@ appVersion: v3.4.8
 name: argo-workflows
 description: A Helm chart for Argo Workflows
 type: application
-version: 0.29.2
+version: 0.29.3
```

Comment:
This will be a breaking change for current minio users, no? If I'm reading this correctly (which there's every chance I'm not!), they will need to rename their secrets?

Reply:
I think @tico24 is right, this PR will affect some users.
I think it should be `0.30.0`.

Reply:
It's better to specify the secret+key name in `values.yaml`, with template expansion support, with the default being the current value.

Reply:
That's the only way it makes sense to structure this PR, IMHO.

Reply:
@vladlosev please review, I've exposed another value for secretName with the default being the current value and used template to parse it, let me know if it fits